Programming_Engine/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
Copyright header needed (as it is a CS file and consistent with our other approaches)